specs/experimental/validator-system-v2/validator-manager.md

```diff
@@ -128,22 +128,38 @@ Only validators with an `ACTIVE` status can submit outputs.
 
 If a validator has not submitted output when it was selected as a priority validator, the validator's
 `noSubmissionCount` is incremented by 1. If the validator submits output the next time it is selected as a priority
-validator, `noSubmissionCount` is reset to 0. Otherwise, if the `noSubmissionCount` exceeds `JAIL_THRESHOLD`, the
+validator, the `noSubmissionCount` is reset to 0. Otherwise, if the `noSubmissionCount` exceeds `JAIL_THRESHOLD`, the
 validator is deemed to have failed to fulfill its obligations. Accordingly, the validator is sent to jail and removed
-from the validator tree. After `JAIL_PERIOD` has elapsed, the validator can be unjailed by calling `tryUnjail` function
-on its own and activated if the activation conditions are met.
+from the validator tree during `SOFT_JAIL_PERIOD`.
+
+Additionally, the validator will be jailed if they submit incorrect outputs or challenge correctly submitted outputs. In
+other words, a loser of challenge will result in being jailed for the `HARD_JAIL_PERIOD`. This `HARD_JAIL_PERIOD` is
+longer than the `SOFT_JAIL_PERIOD` because such actions are considered threatening to the assets on Layer 2.
+
+After each jail period has elapsed, the validator can be unjailed by calling `tryUnjail` function on its own and
+activated if the activation conditions are met.
 
 ## Entry Point
 
 Validator Manager contract acts as an entry point for external contracts to slash challenge losers or distribute output
 rewards. Below is a detailed description of each case.
 
+### Bond
+
+Validator System V2 brings the similar bond mechanism as the previous system. The validator must stake a portion of KRO
+deposited as a bond every time they submit an output or create a challenge. Bond amount for each output submission or
+challenge creation is defined as `BOND_AMOUNT`. If there is not enough deposited KRO to bond, the validator will no
+longer be able to submit outputs or create challenges.
+
 ### Slashing
 
-When the challenge ends, the challenge loser is slashed. To apply the slash to the loser, the
-[Colosseum](../../fault-proof/challenge.md#contract-interface) contract calls Validator Manager, which calls Asset
-Manager to calculate the slash amount and apply the change. The loser is sent to jail and the slash amount is
-accumulated as a pending challenge reward.
+Slashing occurs during the challenge process. The assets subject to slashing are the bonds used for output submission or
```

Comment:
```suggestion
Slashing occurs during the challenge process. The assets subject to slashing are the bonds bonded during output submission or
```

Reply:
How about this? Because the same words are duplicated in the same sentence.

`The assets subject to slashing are the bonds for output submission`